core/src/test/java/org/apache/iceberg/TestRowDelta.java

```diff
@@ -896,17 +896,14 @@ public void testAddDeleteFilesMultipleSpecs() {
     Map<String, String> summary = snapshot.summary();
 
     Assert.assertEquals(
-        "Should change 4 partitions", "4", summary.get(CHANGED_PARTITION_COUNT_PROP));
+        "Should change 3 partitions", "3", summary.get(CHANGED_PARTITION_COUNT_PROP));
     Assert.assertEquals("Should add 1 data file", "1", summary.get(ADDED_FILES_PROP));
     Assert.assertEquals("Should have 4 data files", "4", summary.get(TOTAL_DATA_FILES_PROP));
     Assert.assertEquals("Should add 3 delete files", "3", summary.get(ADDED_DELETE_FILES_PROP));
     Assert.assertEquals("Should have 3 delete files", "3", summary.get(TOTAL_DELETE_FILES_PROP));
     Assert.assertEquals("Should add 3 position deletes", "3", summary.get(ADDED_POS_DELETES_PROP));
     Assert.assertEquals("Should have 3 position deletes", "3", summary.get(TOTAL_POS_DELETES_PROP));
 
-    Assert.assertTrue(
-        "Partition metrics must be correct",
-        summary.get(CHANGED_PARTITION_PREFIX).contains(ADDED_DELETE_FILES_PROP + "=1"));
```

Comment:
this was in fact checking the entry, which we don't produce anymore:
```
 "partitions." -> "added-position-delete-files=1,added-delete-files=1,added-files-size=10,added-position-deletes=1"
```
The summaries map also had the following entries for the 3 partitions:
```
 "partitions.data_bucket=0" -> "added-position-delete-files=1,added-delete-files=1,added-files-size=10,added-position-deletes=1"
 "partitions.data=abc" -> "added-position-delete-files=1,added-delete-files=1,added-files-size=10,added-position-deletes=1"
 "partitions.data=xyz" -> "added-data-files=1,added-records=1,added-files-size=10"
```